xml-parsing-example/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fluent/fluent-bit:1.8-debug
```

Comment:
Why 1.8? Not 1.9, is that the issue with the outbuf?

Reply:
no issue, I simply copied from my existing fluent talk Dockerfile :D